src/styles/App.css

```diff
@@ -1,16 +1,36 @@
 body {
-    background-color: #99dacc;
-    font-family: Helvetica, sans-serif;
+    background-color: #e2e2e2;
   }
   
   .App {
     text-align: center;
     padding-bottom: 85px;
     font-family: 'Poppins', sans-serif;
   }
+
+  .Font {
+    font-family: 'Poppins', sans-serif;
+  }
   
 
   a.Recipe-link {
     color:darkslategrey;
-    padding-top: 75%;
-  }
\ No newline at end of file
+  }
+
+  .container img {
+    width: 100%;
+ }
+
+ .MuiGrid-spacing-xs-1 {
+  width: calc(100% + 8px);
+  margin: -4px;
+  justify-content: center;
```

Comment:
MUI has methods for overriding styles - see: https://mui.com/customization/how-to-customize/#5-global-css-override for details.

This works fine in this example but keep in mind theres different options that exist and you need to know why one way of doing the same thing could be better than another and why.